addons/l10n_cy/data/account_tax_report_data.xml

```diff
@@ -0,0 +1,168 @@
+<?xml version="1.0" encoding="utf-8" ?>
+<odoo auto_sequence="1">
+    <record id="tax_report" model="account.report">
+        <field name="name">Tax Report</field>
+        <field name="root_report_id" ref="account.generic_tax_report"/>
+        <field name="country_id" ref="base.cy"/>
+        <field name="filter_fiscal_position" eval="True"/>
+        <field name="availability_condition">country</field>
+        <field name="column_ids">
+            <record id="tax_report_balance" model="account.report.column">
+                <field name="name">Balance</field>
+                <field name="expression_label">balance</field>
+            </record>
+        </field>
+        <field name="line_ids">
+            <record id="tax_report_line_1" model="account.report.line">
+                <field name="name">1. VAT due on sales and other outputs</field>
+                <field name="code">cy_1</field>
+                <field name="expression_ids">
+                    <record id="tax_report_line_1_amount" model="account.report.expression">
+                        <field name="label">balance</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">1</field>
+                    </record>
+                </field>
+            </record>
+            <record id="tax_report_line_2" model="account.report.line">
+                <field name="name">2. VAT due on acquisitions from other EU Member States</field>
+                <field name="code">cy_2</field>
+                <field name="expression_ids">
+                    <record id="tax_report_line_2_amount" model="account.report.expression">
+                        <field name="label">balance</field>
+                        <field name="engine">tax_tags</field>
+                        <field name="formula">2</field>
+                    </record>
+                </field>
+            </record>
+            <record id="tax_report_line_3" model="account.report.line">
```

Comment:
Like olma said on the task you can add a hierachy_level 0, the line will have a grey background which will be more readable for the user :smile: